.nvmrc

```diff
@@ -1 +1 @@
-v12.16.3
+v12.18.2
```

Comment:
Bumped this because we need a higher version of node for esmodule support with Web Test Runner.